resources/views/users/partials/_form_users.blade.php

```diff
@@ -0,0 +1,41 @@
+@include('layouts.errors')
+
+{{ csrf_field() }}
+
+<div class="form-item -padded">
+    <label class="field-label" for="key">User identification key:</label>
+    <input type="text" name="key" id="key" class="text-field" placeholder="Email, Mobile Phone Number, Northstar ID or Phoenix ID" value="{{ $user->id or old('key') }}"/>
```

Comment:
@DFurnes `{{ $user->id or old('key') }}` worked ;)

@deadlybutter (see above) we figured out a decent way to grab old values stored in session (for example on validation errors and you get returned to the form) or if it's already been set and you're loading the data into the field :smile: